readme.md

```diff
@@ -0,0 +1,113 @@
+# Pico-Bot
+
+Bot para automatizar a criação de stickers a partir de imagens ou mensagens.
+Utilize o comando `/help` para ver os comandos disponíveis.
+
+Código disponível no repositório:
+[http://github.com/caravelahc/pico-bot]()
+
+Feito por [@diogojs](https://t.me/diogojs) e [@caravelahc](https://t.me/caravelahc).
+
+## A) Como Instalar
+
+### A.1) Python
+
+Tenha certeza que possui o Python com versão maior ou igual à 3.7.0. 
+
+### A.2) Gerenciador de pacotes `poetry`
+
+Primeiro de tudo, instale o `poetry`, que é um gerenciador de pacotes e dependências do Python.
+
+### A.3) Dependências
+
+Para instalar as dependências do projeto, vamos criar um ambiente local para que possamos instalar os pacotes sem afetar o sistema:
```

Comment:
`poetry install` already creates a virtualenv:

```console
$ git clone https://github.com/caravelahc/pico-bot
...
$ cd pico-bot 
$ poetry install
Creating virtualenv picobot-py3.7 in /home/jptiz/.cache/pypoetry/virtualenvs
Installing dependencies from lock file


Package operations: 25 installs, 0 updates, 0 removals

  - Installing pycparser (2.19)
  - Installing zipp (0.5.1)
  - Installing asn1crypto (0.24.0)
  - Installing cffi (1.12.3)
  - Installing importlib-metadata (0.17)
  - Installing six (1.12.0)
  - Installing appdirs (1.4.3)
  - Installing atomicwrites (1.3.0)
  - Installing attrs (19.1.0)
  - Installing certifi (2019.3.9)
  - Installing click (7.0)
  - Installing cryptography (2.7)
  - Installing entrypoints (0.3)
  - Installing future (0.17.1)
  - Installing mccabe (0.6.1)
  - Installing more-itertools (7.0.0)
  - Installing pluggy (0.12.0)
  - Installing py (1.8.0)
  - Installing pycodestyle (2.5.0)
  - Installing pyflakes (2.1.1)
  - Installing toml (0.10.0)
  - Installing black (18.9b0)
  - Installing flake8 (3.7.8)
  - Installing pytest (3.10.1)
  - Installing python-telegram-bot (11.1.0)
  - Installing picobot (0.1.0)
$ poetry run which python
/home/jptiz/.cache/pypoetry/virtualenvs/picobot-py3.7/bin/python
```

Reply:
It was not working on my local machine, so I've to create these different steps. Currently I'm using MacOS Mojave 10.14.6, maybe change the readme in order to create setup instructions for different OSs (Linux distros, MacOS, etc.).

Reply:
Oh, thats strange. Can you confirm your `poetry --version`?